### PR TITLE
Feature/cap mirseq

### PR DESCRIPTION
**Pull request by jerowe**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Recipe for cap mirseq
http://bioinformaticstools.mayo.edu/research/cap-mirseq/

### CODE REVIEW

File: recipes/cap-mirseq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - samtools ==0.1.19
+      - picard ==1.97
+      - fastqc ==0.11.5
+      - bedtools ==2.17.0
+      - mirdeep2 ==2.0.0.7
+      - vcftools ==0.1.11
+
+source:
+  fn: cap-mirseq.tar.gz
+  url: http://bioinformaticstools.mayo.edu/research/wp-content/plugins/download.php?url=https://s3-us-west-2.amazonaws.com/mayo-bic-tools/cap-mirna/CAP-miRSEQ.tgz
```
Comment (bgruening) [inline recipes/cap-mirseq/meta.yaml]: Wow, this URL looks scary.  Can you include a checksum? Do you know if this URL is sustainable ... Maybe you can use https://github.com/galaxyproject/cargo-port to make it more stable?
Just a suggestion ....

Comment (jerowe) [inline recipes/cap-mirseq/meta.yaml]: What about now?

**Comment by bgruening** [comment]
Just one question, is there any reason you pinned the version such strict.

**Comment by jerowe** [comment]
Because those are the versions addressed in the cap mrseq papet

**Comment by bgruening** [comment]
I would change them to >= or remove it completely.

**Comment by johanneskoester** [comment]
Please update the branch. We have reverted to our old stable build system.

**Comment by jerowe** [comment]
Why am I getting messages about not being up to date with the base branch all over the place? The build system never used to care if I was missing recipes as long as I didn't have any conflicts...

**Comment by johanneskoester** [comment]
That was kind of my fault. I activated that feature temporarily to ensure that all PRs use the reverted build scripts. I have now deactivated it again. Sorry for the inconvenience.

**Comment by jerowe** [comment]
That's fine. I was just wondering. ;-)

**Pull request merged**